android/src/main/java/com/reactnativereceivesharingintent/ReceiveSharingIntentHelper.java

```diff
@@ -488,8 +488,10 @@ public void clearFileNames(Intent intent) {
     } else if (
       type.startsWith("image") ||
       type.startsWith("video") ||
-      type.startsWith("application")
+      type.startsWith("application") ||
+      type.startsWith("*/*")
```

Comment:
can you tell me what we receive in `type`?

Reply:
When it's only images it's `image/*`, for videos `video/*` etc., and when there's a mix it's `*/*` from what I was able to debug

Reply:
Awesome, can you please add this as a comment on top of that line?